### ISSUE TITLE
Device configuration

### ISSUE THREAD

**Issue by poobaisle** [tananaev/traccar#2130]
As I am not able to find the brand of my tracking device, I would like to configure it to send data to traccar and read the HEX message. However, how do I configure the device ? Is there a generic software to do that ? From the device itself, I have only access to a serial port. Thank you

**Comment by tananaev** [comment]
There is no generic software. Usually devices are configured using specially formatted SMS commands. Format is different for different brands/models. Some devices can be configured via serial port as well, but it's less common option.

**Comment by poobaisle** [comment]
So, unless the company who installed them first give me the formatted SMS commands, I am toasted ! I have 10 of those...

**Comment by tananaev** [comment]
Last hope would be to post some photos on forum or here and see if some other people have same/similar device and know commands.

**Comment by poobaisle** [comment]
I did it already... without success. I'll post it on a local forum maybe i'll be more lucky
[E-series version Eng Brochure.pdf](https://github.com/tananaev/traccar/files/371621/E-series.version.Eng.Brochure.pdf)

**Comment by tananaev** [comment]
Did I miss it? I don't recall seeing it before.

**Comment by poobaisle** [comment]
Well, this is how I got the information about the HEX file to find which protocol it is using. About this device, what if I still have other cars currently using it with the same company. From the web based application, can I see somewhere which port is listening to the device and which protocol is being used ? Maybe too complex... I'd better find someone on the internet who uses the very same devices and can help me...

**Comment by tananaev** [comment]
If the application is not hosted on your own server, I don't think you can find anything about protocol.

**Comment by poobaisle** [comment]
Yes, I agree. Social engineering is the best option to help me out

**Comment by poobaisle** [comment]
I don't know if I can get anything from these additional information ? Apparently the module of the tracking device is Wavecom Q2687.
![swr](https://cloud.githubusercontent.com/assets/20451582/16978261/4a286b3e-4e85-11e6-8bea-d81f7e0ae8ad.JPG)
I downloaded of brochure about this module attached (don't know it is said to be confidential since it is on the web).
[Q2687_Wireless_CPU_Product_Technical_Specification-Rev005.pdf](https://github.com/tananaev/traccar/files/373041/Q2687_Wireless_CPU_Product_Technical_Specification-Rev005.pdf)

**Comment by poobaisle** [comment]
Will I be able to configure the tracking device if I download one of the tool here ?
http://www.smssolutions.net/hardware/gsmgprs2/2wavecomq2687/

**Comment by Abyss777** [comment]
Hello, @poobaisle
Try another tack, If you have access to installed server you can try to dump network traffic from devices on it.

**Comment by poobaisle** [comment]
What do you mean @Abyss777  ? How ?

**Comment by Abyss777** [comment]
@poobaisle 
> what if I still have other cars currently using it with the same company. From the web based application...

If this application installed on the own company server you can talk to system administrator and try dump traffic to this server, with help of `wireshark` or `tcpdump` (depends on system)

**Comment by poobaisle** [comment]
Wireshark... Yes, I can try to switch on 1 tracking device and see what information I gather maybe...
